AC_e-Reader_Card_Creator/CCC-Frontend.cs

```diff
@@ -593,6 +632,332 @@ private void ToggleDarkMode(object sender, EventArgs e)
                     }
                 }
             }
+
+            SaveDarkModePreference(isDarkModeEnabled);
+        }
+
+        private void ApplyDarkMode()
+        {
+            BackColor = isDarkModeEnabled ? Color.FromArgb(40, 40, 40) : SystemColors.Control;
+
+            foreach (Control control in Controls)
+            {
+                if (control is ToolStrip)
+                {
+                    continue;
+                }
+
+                if (isDarkModeEnabled)
+                {
+                    control.BackColor = Color.FromArgb(40, 40, 40);
+                    control.ForeColor = Color.White;
+
+                    if (control is TextBox textBox)
+                    {
+                        textBox.BackColor = Color.FromArgb(31, 31, 31);
+                        textBox.BorderStyle = BorderStyle.FixedSingle;
+                    }
+                    else if (control is Button button)
+                    {
+                        button.FlatStyle = FlatStyle.Flat;
+                        button.FlatAppearance.BorderColor = Color.White;
+                    }
+                    else if (control is ComboBox comboBox)
+                    {
+                        comboBox.BackColor = SystemColors.Window;
+                        comboBox.ForeColor = SystemColors.ControlText;
+                    }
+                }
+                else
+                {
+                    control.BackColor = SystemColors.Control;
+                    control.ForeColor = SystemColors.ControlText;
+
+                    if (control is TextBox textBox)
+                    {
+                        textBox.BackColor = Color.White;
+                        textBox.BorderStyle = BorderStyle.Fixed3D;
+                    }
+                    else if (control is Button button)
+                    {
+                        button.BackColor = Color.White;
+                        button.FlatStyle = FlatStyle.Standard;
+                        button.FlatAppearance.BorderColor = Color.White;
+                    }
+                    else if (control is ComboBox comboBox)
+                    {
+                        comboBox.BackColor = SystemColors.Window;
+                        comboBox.ForeColor = SystemColors.ControlText;
+                    }
+                }
+            }
+        }
+
+        private void SaveDarkModePreference(bool darkModeEnabled)
+        {
+            string appDataFolderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "AC e-Reader Card Creator");
```

Comment:
I suggest renaming the created directory and file as `e-ReaderCardCreator/config.txt`. Inside `config.txt`, we could have a `darkModePreference:` variable which can be read by the app. This way, we can add other variables to the config file if we add other user-specific settings in the future.